lib/signaling/v2/peerconnectionmanager.js

```diff
@@ -25,12 +26,31 @@ function PeerConnectionManager(iceServerSource, options) {
   QueueingEventEmitter.call(this);
 
   options = Object.assign({
-    audioContext: audioContext,
+    AudioContextFactory: require('../../webaudio/audiocontext'),
     MediaStream: MediaStream,
     PeerConnectionV2: PeerConnectionV2
   }, options);
 
+  // NOTE(mroberts): By default, only construct an AudioContext (and therefore
+  // trigger the "dummy" MediaStreamTrack behavior) in Firefox.
+  var audioContext = options.audioContext ? options.audioContext : null;
```

Comment:
@markandrus Shouldn't this be `var audioContext = options.AudioContextFactory.getOrCreate(this);` since you've converted `audiocontext.js` to a factory ?